config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'ramens#top'
-  get 'ramens/index', to: 'ramens#index'
-  get 'ramens/:id', to: 'ramens#show'
-  
+  root to: 'ramen_stores#top'
+
+  get 'ramens/index', to: 'ramen_stores#index'
```

Comment:
一覧の場合は下記でもわかるかなと思います！
```suggestion
  get 'ramens', to: 'ramen_stores#index'
```